gips/test/sys/expected/sentinel2.py

```diff
@@ -111,29 +111,31 @@
    vari        Visible Atmospherically Resistant Index 
 \x1b[1m
 Standard Products
-\x1b[0m   ref         Surface reflectance                     
+\x1b[0m   rad         Surface-leaving radiance                
+   ref         Surface reflectance                     
 """
 }
 
 
 t_project = {
+    'compare_stderr': False,
     'created': {
-        '0': None,
```

Comment:
note both red and green values are wrong; `gips_project` seems to generate nothing but NaN values in both this branch and the prior 200 branch.